src/components/RadioGroup/RadioGroup.scss

```diff
@@ -0,0 +1,3 @@
+.moonstone-radioGroup {
+    width: max-content;
```

Comment:
Why do we need that?

Reply:
no need, I forgot to delete it, thanks :-)